Include/httpClient/httpClient.h

```diff
@@ -783,19 +398,8 @@ HCHttpCallRequestGetTimeout(
 /// <param name="responseString">the response body string of the HTTP call</param>
 HC_API void HC_CALLING_CONV
 HCHttpCallResponseGetResponseString(
-    _In_ HC_CALL_HANDLE call,
-    _Out_ PCSTR_T* responseString
-    );
-
-/// <summary>
-/// Set the response body string of the HTTP call
-/// </summary>
-/// <param name="call">The handle of the HTTP call</param>
-/// <param name="responseString">the response body string of the HTTP call</param>
-HC_API void HC_CALLING_CONV
-HCHttpCallResponseSetResponseString(
-    _In_ HC_CALL_HANDLE call,
-    _In_ PCSTR_T responseString
+    _In_opt_ HC_CALL_HANDLE call,
```

Comment:
Why is this optional? (also applies to the other getters below)

Reply:
As doc'd in header if null, then it sets default.

Reply:
There is a default response string?

I guess that can make sense, seems a bit weird though

Reply:
Oh right, this one shouldn't be _In_opt_